src/searchByUser.php

```diff
@@ -4,6 +4,7 @@
     // $js = "scripts/attraction.js";
     $fluid_container = true;
     $load_datatable_libraries = true;
```

Comment:
if we don't use $load_datatable_libraries anymore, we should remove it from here. However if it is used in other files, where buttons are not needed, we should not combine libraries and buttons under one variable.